lib/rmt/config.rb

```diff
@@ -3,15 +3,34 @@
 
 Config.setup do |config|
   config.merge_nil_values = false
+
+  unless ENV['RAILS_ENV'] == 'test'
+    config.schema do
+      required(:scc).schema do
+        required(:username).filled
+        required(:password).filled
+      end
+    end
+  end
 end
 
 Config.load_and_set_settings(
+  File.join(__dir__, '../../config/default_rmt.yml'),
   '/etc/rmt.conf',
   File.join(__dir__, '../../config/rmt.yml'),
   File.join(__dir__, '../../config/rmt.local.yml')
 )
 
 module RMT::Config
+  def self.method_missing(key)
+    raise NoMethodError if Settings[key].nil?
```

Comment:
`try` guaranteed safe navigation, this actually might raise exceptions. Although there's probably no way to address this.

Reply:
I've changed this implementation a bit because `rubocop` was complaining about the lack of falling back to `super` (https://github.com/rubocop-hq/ruby-style-guide#no-method-missing), but the results are the same.